--- origsrc/libtsmux-0.2.0/tsmux/Makefile.am	2006-11-16 09:52:08.000000000 -0600
+++ src/libtsmux-0.2.0/tsmux/Makefile.am	2007-03-22 22:10:25.234375000 -0500
@@ -6,7 +6,7 @@
 
 libtsmux_la_CFLAGS = $(GLIB_CFLAGS) -I\$(top_srcdir) -I\$(top_builddir)
 libtsmux_la_LIBADD = $(GLIB_LIBS)
-libtsmux_la_LDFLAGS = 
+libtsmux_la_LDFLAGS = -no-undefined
 
 # headers we need but don't want installed
 noinst_HEADERS = crc.h
